src/screens/Login/ChooseAccountForm.tsx

```diff
@@ -38,15 +39,22 @@ export const ChooseAccountForm = ({
           setShowLoggedOut(false)
           Toast.show(_(msg`Already signed in as @${account.handle}`))
         } else {
-          await initSession(account)
-          logEvent('account:loggedIn', {
-            logContext: 'ChooseAccountForm',
-            withPassword: false,
-          })
-          track('Sign In', {resumedSession: true})
-          setTimeout(() => {
-            Toast.show(_(msg`Signed in as @${account.handle}`))
-          }, 100)
+          try {
+            await initSession(account)
+            logEvent('account:loggedIn', {
+              logContext: 'ChooseAccountForm',
+              withPassword: false,
+            })
+            track('Sign In', {resumedSession: true})
+            setTimeout(() => {
+              Toast.show(_(msg`Signed in as @${account.handle}`))
+            }, 100)
+          } catch (e: any) {
+            logger.error('choose account: initSession failed', {
+              message: e.message,
+            })
+            onSelectAccount(account)
```

Comment:
This is the main change here. It ensures that if there's an error from `initSession` (which, in my understanding, technically can't happen yet — but we could propagate errors in the future properly, which would active this codepath). If this codepath starts firing in the future, an error during `initSession` will now preselect the account in which you're trying to log in (and ask you to enter the password). Previously, you'd silently stay on the previous "select an account" form with no visual feedback about the failure.

(I'm actually not sure this is better. It's not ideal that we kick you to log in even if the problem isn't solved by a password — e.g. if it's a network issue. But let's capture the intent of who you were trying to log in as, and show some visual feedback.)

We know this state exists so we can iterate on it later.